gradle.properties

```diff
@@ -3,18 +3,10 @@ org.gradle.caching=true
 org.gradle.jvmargs=-XX:+UseParallelGC -Xmx4g -Dfile.encoding=UTF-8 -XX:MaxMetaspaceSize=2g -XX:+HeapDumpOnOutOfMemoryError -Xms256m
 # use parallel execution
 org.gradle.parallel=true
-# https://docs.gradle.org/7.6/userguide/configuration_cache.html
-org.gradle.unsafe.configuration-cache=true
-
-# https://blog.jetbrains.com/kotlin/2022/07/a-new-approach-to-incremental-compilation-in-kotlin/
-kotlin.incremental.useClasspathSnapshot=true
+# https://docs.gradle.org/8.1/userguide/configuration_cache.html
+org.gradle.configuration-cache=true
 
 # Disable buildFeatures flags by default
-android.defaults.buildfeatures.aidl=false
-android.defaults.buildfeatures.buildconfig=false
-android.defaults.buildfeatures.renderscript=false
```

Comment:
Don't these defaults change with AGP 8? We're still using 7.4.2. Or am I misreading the docs?

Reply:
Oooops, I forget that we are still on AGP 7.